Dockerfile

```diff
@@ -191,9 +191,8 @@ ENV INSTALL_MYSQL_CLIENT=${INSTALL_MYSQL_CLIENT} \
 
 # Only copy install_mysql.sh to not invalidate cache on other script changes
 COPY scripts/docker/install_mysql.sh /scripts/docker/install_mysql.sh
-RUN bash ./scripts/docker/install_mysql.sh dev
-
-RUN mkdir -p /root/.local/bin
+RUN bash ./scripts/docker/install_mysql.sh dev && \
+    mkdir -p /root/.local/bin
```

Comment:
Is it related to Python or MySQL? If Python, then we can move that to the 204 line.

Reply:
Yep. might be better there.